bedrock/firefox/forms.py

```diff
@@ -10,8 +10,6 @@
 
 from lib.l10n_utils.dotlang import _lazy as _
 
-LANG_FILES = ['firefox/whatsnew-fx37']
-
```

Comment:
I'd suggest leaving the `invalid` string as-is, but adding a comment stating that it's not actually in use or localized, but is left in as a placeholder for potential future use. How's that sound?